doc/pitfalls/data_leakage.rst

```diff
@@ -0,0 +1,150 @@
+.. _data_leakage:
+
+============
+Data leakage
+============
+
+Data leakage occurs when information that would not be available at prediction
+time, is used when building the model. This results in overly optimstic
+performance estimates, for example from :ref:`cross validation
+<cross_validation>`, but poorer performance when the model is asked to predict
```

Comment:
Maybe
```suggestion
<cross_validation>`, but poorer performance when the model is used
```
since it's not sentient yet ) Though maybe I'm misinterpreting the wording.